module/data/abstract.mjs

```diff
@@ -440,17 +440,15 @@ export class ItemDataModel extends SystemDataModel {
    * @typedef {SystemDataModelMetadata} ItemDataModelMetadata
    * @property {boolean} enchantable    Can this item be modified by enchantment effects?
    * @property {boolean} hasEffects     Display the effects tab on this item's sheet.
-   * @property {boolean} inventoryItem  Should this item be listed with an actor's inventory?
-   * @property {number} inventoryOrder  Order this item appears in the actor's inventory, smaller numbers are earlier.
    * @property {boolean} singleton      Should only a single item of this type be allowed on an actor?
+   * @property {InventorySectionDescriptor} [inventory]  Configuration for displaying this item type in its own section
+   *                                                     in creature inventories.
    */
 
   /** @type {ItemDataModelMetadata} */
   static metadata = Object.freeze(foundry.utils.mergeObject(super.metadata, {
     enchantable: false,
     hasEffects: false,
-    inventoryItem: false,
```

Comment:
Would probably be good to have some sort of deprecation period for this change. Just in my quick testing with Tasha's this caused the tattoo items to appear in the wrong tab and be hard to find in the Compendium Browser.

Reply:
Factored this out to `ItemDataModel.inventorySection` getter and added a deprecation path for `ItemDataModel.metadata.inventoryItem`.